client/templates/globals/connectioninfo.js

```diff
@@ -8,6 +10,12 @@ Template.connectionInfo.helpers({
     connectionWaitTime() {
         var secondsToRetry = (Meteor.status().retryTime - (new Date()).getTime())/1000;
```

Comment:
It would be nice if you could change this `var` keyword to `const`, too. Even if it was not your line of code ;-)